### DISCUSSION TITLE
Call another react context in zustand store

### DISCUSSION THREAD

**Discussion by majeeddl** [pmndrs/zustand#1570] [Q&A]
I wanna to call another react context in zustand's events, I see the error that we can not use that, is there any solution for this situation?

**Comment by dai-shi** [comment]
Yeah, you can't access to context values from outside global store. Maybe, you want some kind of pub/sub mechanism?

**Reply by majeeddl** [reply]
yes , what is your idea when we want to  access Context Provider ? could you show me an example of  pub/sub mechanism?

**Reply by dai-shi** [reply]
global can't access context, but context can access global. so from context, you add something (depending on your need) to a global store.

But, why do you want it in the first place? It sounds like the context approach is wrong, and you would just do everything in a global store.
